components/sso-hostobject/org.wso2.carbon.hostobjects.sso/src/main/java/org/wso2/carbon/hostobjects/sso/internal/builder/AuthReqBuilder.java

```diff
@@ -180,11 +179,15 @@ private static Issuer buildIssuer(String issuerId) {
      *
      * @return NameIDPolicy object
      */
-    private static NameIDPolicy buildNameIDPolicy() {
-        NameIDPolicy nameIDPolicy = new NameIDPolicyBuilder().buildObject();
-        nameIDPolicy.setFormat(SSOConstants.SAML2_NAME_ID_POLICY);
-        nameIDPolicy.setAllowCreate(true);
-        return nameIDPolicy;
+    private static NameIDPolicy buildNameIDPolicy(String nameIdPolicy) {
+        NameIDPolicy nameIDPolicyObj = new NameIDPolicyBuilder().buildObject();
```

Comment:
Hi,

It is not the same code. Other piece of code builds a NameID and here it builds a NameIDPolicy. Anyway better to move both methods to Util class. Will do it. Thanks for pointing out.